### PR TITLE
Fixes lung swapping giving you false air alerts and slightly improves their descriptions

### LINKED ISSUE #29360: Lung Transplants and Alerts
This existed prior to my refactor of lungs; my refactor most definitely did not cause it.

When you perform a lung transplant from one type to another (say, giving a plasmaman normal lungs), it causes issues with alerts.

More specifically, the plasmaman receiving the lungs will be able to breathe oxygen as normal and not taken any suffocation damage....that said, he will still have an alert informing him that he needs to breathe in plasma.

This becomes even more messy and convoluted if you have multiple different types of lungs that all breathe different gasses. Needless to say, you can wind up with a huge number of alerts on your screen, but you're not taking any damage, nor should they be displayed.

### PR DESCRIPTION
**Pull request by RikuTheKiller**
## About The Pull Request

Title, primarily fixes plasmamen getting oxygen alerts when they get their lungs changed.

fixes #29360

## Why It's Good For The Game

Bug fix good. Messages being consistent good.

## Changelog

:cl:
fix: Air alerts no longer prescribe you with breathing things you'll just die from.
qol: Air alerts now tell you that you've got a plasma tank in your survival box if you're a plasmaman and don't say you have a gas mask in there, it's a breath mask.
/:cl:

### CODE REVIEW

**Comment by RikuTheKiller** [comment]
FUCK I OPENED IT WITHOUT A DESCRIPTION

**Comment by Mooshimi** [comment]
Please link this with #29360 if it fixes it.

**Comment by RikuTheKiller** [comment]
Will do!

File: code/modules/surgery/organs/lungs.dm [hunk 1]
```diff
@@ -88,6 +88,15 @@
 
 	var/crit_stabilizing_reagent = /datum/reagent/medicine/epinephrine
 
+///Simply exists so that you don't keep any alerts from your previous lack of lungs.
+/obj/item/organ/internal/lungs/Insert(mob/living/carbon/receiver, special = FALSE, drop_if_replaced = TRUE)
+	receiver.clear_alert(ALERT_NOT_ENOUGH_OXYGEN)
+	receiver.clear_alert(ALERT_NOT_ENOUGH_CO2)
```
Comment (Mothblocks) [inline code/modules/surgery/organs/lungs.dm:94]: This gives me pause. Is there a case where you would have these alerts beforehand, get new lungs, and they would still be relevant? Are they going to clear, delay, then add themselves back?

Comment (RikuTheKiller) [reply code/modules/surgery/organs/lungs.dm:94]: Hmm? They would add themselves back almost immediately, pretty sure. I might have to test that though.

Comment (RikuTheKiller) [reply code/modules/surgery/organs/lungs.dm:94]: I tested it, it gave it back to me less than a second after.

Comment (Mothblocks) [reply code/modules/surgery/organs/lungs.dm:94]: Alright. I was concerned about the UX but if it's less than a second then I suppose that's fine

**Pull request merged**